packages/ui5-info/src/ui5-version-info.ts

```diff
@@ -165,10 +167,10 @@ async function parseUI5VersionsOverview(): Promise<UI5VersionOverview[]> {
  * @returns Array of UI5 versions
  */
 const retrieveUI5VersionsCache = async (
-    type: ui5VersionsType.official | ui5VersionsType.snapshot | ui5VersionsType.overview,
+    type: ui5VersionsType.official | ui5VersionsType.snapshot | ui5VersionsType.support,
     useCache = false,
     snapshotUrl?: string
-): Promise<string[] | UI5VersionOverview[]> => {
+): Promise<string[] | UI5Version[]> => {
     if (!useCache) {
         switch (type) {
             case ui5VersionsType.official:
```

Comment:
```suggestion
            case ui5VersionsType.official:
            case ui5VersionsType.support:
```
Shouldn't we just merge official and support as they are both making the same API call. 
Also, I think in some flows require both data and should now be able to get the data using a single HTTP call instead of the 2 previously.

Reply:
Agreed. 
Initially, I tried to take the least invasive approach to reduce the risk of regressions, but now that we're making a call to the same API, it makes sense to consolidate.
Implemented in [2062785](https://github.com/SAP/open-ux-tools/pull/2189/commits/20627858a7f146eb6dabc79f8d01c4dad53546c5).
Thanks @devinea